clickhouse_std.go

```diff
@@ -272,8 +293,12 @@ func (std *nativeTransport) PrepareContext(ctx context.Context, query string) (d
 
 func (std *nativeTransport) Close() error { return std.conn.close() }
 
+type IBatch interface {
```

Comment:
I'm curious if these interface introductions increase allocations and impact performance? Id like to see before and after comparisons if possible.

Reply:
i.e. its impact on the native interface.

Reply:
Do you have measuring tests?

Reply:
Found benchmark dir

Reply:
presumably similar now we have moved the interface up to the stdBatch?